<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Supported accounts:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_accounts">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lb_error">
     <property name="text">
      <string>There is no supported accounts!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_settings">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Account Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_mail">
        <property name="toolTip">
         <string> If set to &quot;true&quot;, the server will send
 notifications of unread email.</string>
        </property>
        <property name="text">
         <string>Mail notifications</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_archiving">
        <property name="toolTip">
         <string> If set to &quot;true&quot;, the server will save
 Google Talk chats on the server;
 if set to &quot;false&quot;, no conversations
 will be logged.</string>
        </property>
        <property name="text">
         <string>Message archiving</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_suggestions">
        <property name="toolTip">
         <string> If set to &quot;true&quot;, the server will
 automatically add people emailed
 often to the user's roster.</string>
        </property>
        <property name="text">
         <string>Auto-accept suggestions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_shared_statuses">
        <property name="toolTip">
         <string> Enable multiple applications signed in
 as the same user to report the same
 status message. </string>
        </property>
        <property name="text">
         <string>Enable Shared Statuses</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_nosave">
        <property name="toolTip">
         <string> Google Talk server disables server-side
 logging of individual conversations and
 sends notifications.
 Enable &quot;Message archiving&quot; option for use!</string>
        </property>
        <property name="text">
         <string>Enable Off the Record Chats</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_mail_settings">
     <property name="toolTip">
      <string> Enable &quot;Mail notifications&quot; option!</string>
     </property>
     <property name="title">
      <string>Mail Notifications Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rb_all_messages">
        <property name="text">
         <string>Show all unread messages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_new_messages">
        <property name="text">
         <string>Show new messages only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Message:
%1 - name
%2 - e-mail
%3 - message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="te_message"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Show popups</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_interval">
       <property name="minimum">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>seconds (0=disabled, -1=infinite)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lb_wiki">
     <property name="text">
      <string>&lt;a href=&quot;http://psi-plus.com/wiki/plugins#gmail_service_plugin&quot;&gt;Wiki (Online)&lt;/a&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_mail</sender>
   <signal>toggled(bool)</signal>
   <receiver>gb_mail_settings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_archiving</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_nosave</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
